aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -187,6 +197,121 @@ impl MoveHarness {
         self.run(txn)
     }
 
+    pub fn setup_staking(&mut self, account: &Account, initial_stake_amount: u64) {
```

Comment:
I thought this landed already as I approved this but now I seem to get a 2nd chance to ask for something here: I suggest we move this into a different struct and module. One problem with the old infra is that it _a lot_ of stuff and was hard to navigate. Lets try to keep this small and structured. New usage could be like this:

```
let mut stake_harness = StakeHarness::new(harness); // move harness into stake_harness.
stake_harness.setup_staking(...);
stake_harness.harness.some_base_func(...); // can still use functions in the underlying harness
...
let mut harness = stake_harness.into_inner() 
```

Reply:
> 

Another advantage is that you can start adding state specific to the functions in `StakeHarness`. Whether StakeHarness is the right name I don't know. Perhaps `AptosHarness` is better.

Reply:
Updated. I separated out the stake calls into a separate library in src/stake.rs. They don't need to store any state between calls for now so doesn't need to be an object like MoveHarness.